trace/tests/system/flask/main.py

```diff
@@ -38,8 +38,11 @@
 
 app = flask.Flask(__name__)
 
-# Enable tracing, send traces to Stackdriver Trace
-exporter = stackdriver_exporter.StackdriverExporter()
+# Enable tracing, send traces to Stackdriver Trace using background thread transport
+from opencensus.trace.exporters.transports import background_thread
+exporter = stackdriver_exporter.StackdriverExporter(
+    transport=background_thread.BackgroundThreadTransport)
+
```

Comment:
To check my understanding, the flask test uses async transport and the django test uses sync transport?  I think this is a good idea, but we should add a comment here and in the django test explaining that they are intentionally different to test both code paths.

Reply:
Yes correct. Comment added.